### ISSUE TITLE
grab_logits windows only?

### ISSUE THREAD

**Issue by kunwon1** [wbrown/novelai-research-tool#9]
Hi, I'm having a little trouble understanding how logits work. I'm running this on linux, I have grab_logits.exe but no linux binary, and there's no grab_logits.go that I can see, so I don't know how to build a linux version. Am I missing something? (logit_client works fine, so I'm guessing I'm just missing some build command that I'm not familiar with)
